Tasks/find.js

```diff
@@ -3,8 +3,8 @@
 
 const find = (object, ...rest) => {
```

Comment:
I think, the name of the function is low understandable. Consider changing name, for examle: getKey or findKey.